### DISCUSSION TITLE
Ceremonias - Review y Retrospective Meeting + Anuncio Urgente! 26/06/2022 - Sprint 2

### DISCUSSION THREAD

**Discussion by Carolina-Bolatti** [ispc-programador2022/a2g3#35] [Mostrar y contar]
Review y Retrospective Meeting realizado en el día 26/06/2022 en donde, como SM ( @Carolina-Bolatti ), pedí que armemos una reunión de Urgencia para restructurarnos y rearmarnos. 

El 23/06/22 se hizo efectivo el cambio de titularidad de profesores con los de los jueves, Profesores Hector y Silvia Perotti, y se nos indicó que teníamos una semana para realizar el proyecto en Trello. No solo hicimos las ceremonias de cierre del Sprint 2, sino que decidimos pausar hasta nuevo aviso el proyecto de GitHub, si es que se continuaría, dependiendo si es necesario y requerido éste proyecto o no, y si nos indican hacer uno más sencillo en GitHub siendo que la desorganización desde el Politecnico generó mucha perdida de tiempo productivo en el equipo y el alumnado. A su vez, nos organizamos para poner manos a la obra con el proyecto en Trello (Plataforma que ninguno sabía utilizar y estabamos intentando aprender rápidamente) y asignamos un nuevo PO y SM para ese Proyecto, decidimos tareas y asignaciones, etc. 

Pese a todo nos esforzamos mucho como equipo y como alumnos por intentar sobreponernos de la situación compleja que se nos presentó.  

@Carolina-Bolatti @Raimundo-ISPC @emilianoblancodev @Franck-2002 @lapi-rula @BlackmouthOscar @Estannoy @Daniel12fec @CandelaBORDI24 @AlphaAlejandro

![1](https://user-images.githubusercontent.com/105814132/176597056-1e75c320-b7ff-4c2f-a88d-2536dfa1ff25.jpeg)

**Comment by Estannoy** [comment]
Coincido totalmente con respecto a la desorganización del politécnico. Pero como dicen por ahí, lo que no te mata te fortalece. Creo que salimos fortalecidos individualmente y como grupo. Fuimos capaces de llevar adelante un proyecto desde cero en una plataforma que no conocíamos. Creo que cumplimos sobradamente con los objetivos que se nos plantearon a pesar de las dificultades. Se formó un gran equipo y estoy orgulloso de formar parte.
